torchani/models.py

```diff
@@ -76,6 +79,14 @@ def get_resource(file_path):
 
         info_file = get_resource(info_file_path)
 
+        # download resources if not already exist
+        if not os.path.isfile(info_file):
```

Comment:
Could you also check if `~/.local/torchani` contains this file, and if yes, use it?